File: Springboot Code/AgroRent/src/main/java/com/majorproj/agrorent/dto/FarmerResponseDto.java
package com.majorproj.agrorent.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class FarmerResponseDto {
	 private String firstName;
	 private String lastName;
	 private String email;
	 private String role; 
	 private boolean active;
}

File: Dot Net Code/AgroRent/DTOs/FarmerResponseDto.cs
namespace AgroRent.DTOs
{
    public class FarmerResponseDto
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Role { get; set; } = string.Empty;
        public bool Active { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime UpdatedOn { get; set; }
    }
}
